File: NetBeansProjects/RubyChess/test/chess/shade_test.rb
$:.unshift File.join(File.dirname(__FILE__),'..','lib')

require 'test/unit'
require 'chess'

module Chess
  class ShadeTest < Test::Unit::TestCase

    def test_values
      assert_equal(
        [
          Shade::Light,
          Shade::Dark
        ],
        Shade.values
      )
    end
  end
end

File: NetBeansProjects/JavaChess/test/vji/chess/ShadeTest.java
package vji.chess;

import org.junit.Test;
import static org.junit.Assert.*;

/**
 *
 * @author vji
 */
public class ShadeTest {

    @Test
    public void testValues() {
        System.out.println("values");
        Shade[] expected = new Shade[]{
            Shade.Light,
            Shade.Dark
        };
        assertArrayEquals(expected, Shade.values);
    }

}